### ISSUE TITLE
Minor Changes

### ISSUE THREAD

**Issue by mabotkin** [tjsct/tjioi#2]
A few minor changes.
1. Add Facebook as a sponsor!
2. Reorganize Contest page as to not have a "Coming Soon" for the current contest, and make a separate page for that.
3. Add 2013 problems.
4. Team size limit still three teams?
5. Update "Northern Virginia Area" in FAQ to just any school?
6. Remove "light breakfast if sufficient funds", and add something about if they have dietary restrictions, they can contact us.
7. Mention T-Shirts in FAQ now that it is confirmed
8. Mention registration fee somewhere, probably FAQ

**Comment by gengkev** [comment]
also, remove Python 2

**Comment by mabotkin** [comment]
excuse me

**Comment by wuxishy** [comment]
@mabotkin I don't think it is worthwhile to add Python2 support at this point because the grader recognize the language through file extension.
Plus UVA only allows Python3, so I don't feel bad for not including 2.

**Comment by mabotkin** [comment]
Also, the current wording says that we will provide computers - can we reword it to make it clear that using our computers is mandatory?

**Comment by gengkev** [comment]
I just updated a bunch of content (see FAQ and About docs in Website folder). I think the only thing left is
* [x] Add Facebook and AoPS as sponsors
